.storybook/main.js

```diff
@@ -1,11 +1,22 @@
 module.exports = {
-  "stories": ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
-  "addons": ["@storybook/addon-links", "@storybook/addon-essentials", "@storybook/addon-interactions"],
+  "stories": ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
```

Comment:
Should this first entry still be: `"../src/**/*.stories.mdx"`?

Reply:
The .mdx extension itself is enough to indicate that the file contains both stories and Markdown content. During the upgrade, the one file - Introduction.stories.mdx - was auto converted to Introduction.mdx.

Reply:
Sounds good.  Thanks for the explanation.